index.html

```diff
@@ -48,186 +59,220 @@ <h3><a href="#">LET'S CONNECT</a></h3>
+            </div>
+        </section>
+    
+        <section class="work-section">
+            <div class="work-section1">
+                <img src="image-resources/Portfolio.png" alt="">
+            </div>
+            <div class="work-section2">
+                <div class="work-imgs card3">
+                    <div class="work-bg"></div>
+                </div>
+                <div class="work-topic">
+                    <h2>Facebook 360</h2>
+                </div>
+                <div class="work-details">
+                    <ul>
+                        <li>Facebook</li>
+                        <li class="work-2nd">∘ Full Stack Dev</li>
+                        <li class="work-2nd">∘ 2015</li>
+                    </ul>
+                </div>
+                <div class="work-paragraph">
+                    <p>Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
+                </div>
+                <div class="work-langs">
+                     <ul>
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javascript</a></li>
+                    <li class="ruby"><a href="#">Ruby on Rails</a></li>
+                     </ul>
+                </div>
+                <div class="work-button">
+                    <button>See Project</button>
+                </div>
+            </div>
+        </section>
+    
+        <section class="work-section">
+            <div class="work-section3">
+                <div class="work-imgs card4">
+                    <div class="work-bg"></div>
+                </div>
+                <div class="work-topic">
+                    <h2>Uber Navigation</h2>
+                </div>
+                <div class="work-details">
+                    <ul>
+                        <li>Uber</li>
+                        <li class="work-2nd">∘ Lead Developer</li>
+                        <li class="work-2nd">∘ 2015</li>
+                    </ul>
+                </div>
+                <div class="work-paragraph">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work-langs">
+                     <ul>
+                    <li><a href="#">html</a></li>
+                    <li class="ruby"><a href="#">Ruby on rails</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javascript</a></li>
+                     </ul>
+                </div>
+                <div class="work-button">
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="work-section4">
+                <img src="image-resources/Snapshoot Portfolio3.jpg" alt="">
+            </div>
+        </section>
     </section>
 
```

Comment:
- [ OPTIONAL]I just noticed that our pointer should be like as is it on the Figma design, Kindly fix this for your project to look more attractive so recruiters can reach you :wink: 
- 
![Screenshot from 2023-02-23 15-28-24](https://user-images.githubusercontent.com/88894525/220920875-68df1a4b-5ecf-40f2-8ddf-0260d92541c3.png)